docs/bagit/readme.txt

```diff
@@ -0,0 +1,51 @@
+This zip file contains a HydroShare resource in Bagit format.  
```

Comment:
should we then list that file in the readme.txt file where the bag structure is shown?